examples/diabetes_example/run_diabetes.py

```diff
@@ -0,0 +1,510 @@
+    type="csv",
+    data_opener=assets_directory / "dataset" / "diabetes_opener.py",
+    description=data_path / "description.md",
+    permissions=permissions,
+    logs_permission=permissions,
+)
+
+# We register the dataset for each of the organisations
+dataset_keys = {
+    client_id: clients[client_id].add_dataset(dataset)
+    for client_id in DATA_PROVIDER_ORGS_ID
+}
+
+for client_id, key in dataset_keys.items():
+    print(f"Dataset key for {client_id}: {key}")
+
+
+# %%
+# The dataset object itself is an empty shell. Data samples are needed in order to add actual data.
+# A data sample contains subfolders containing a single data file like a CSV and the key identifying
+# the dataset it is linked to.
+#
+
+datasample_keys = {
+    org_id: clients[org_id].add_data_sample(
+        DataSampleSpec(
+            data_manager_keys=[dataset_keys[org_id]],
+            test_only=False,
+            path=data_path / f"org_{i + 1}",
+        ),
+        local=True,
+    )
+    for i, org_id in enumerate(DATA_PROVIDER_ORGS_ID)
+}
+
+# %%
+# The data has now been added as an asset through the data samples.
+#
+
+# %%
+# Adding functions to execute with Substra
+# ========================================
+# A :ref:`Substra function<documentation/references/sdk_schemas:FunctionSpec>`
+# specifies the function to apply to a dataset or the function to aggregate models (artifacts).
+# Concretely, a function corresponds to an archive (tar or zip file) containing:
+#
+# - One or more Python scripts that implement the function.
+# - A Dockerfile on which the user can specify the required dependencies of the Python scripts.
+#   This Dockerfile also specifies the function name to execute.
+#
+# In this example, we will:
+#
+# 1. compute prerequisites for first-moment statistics on each data node;
+# 2. aggregate these values on the analytics computation node to get aggregated statistics;
+# 3. send these aggregated values to the data nodes, in order to compute second-moment prerequisite values;
+# 4. finally, aggregate these values to get second-moment aggregated statistics.
+#
+
+
+# %%
+# Local step: computing first order statistic moments
+# ---------------------------------------------------
+# First, we will compute on each data node some aggregated values: number of samples, sum of each numerical column
+# (it will be used to compute the mean), and counts for each category for the categorical column (*Sex*).
+#
+# The computation is implemented in a *Python function* in the `federated_analytics_functions.py` file.
+# We also write a `Dockerfile` to define the entrypoint, and we wrap everything in a Substra ``FunctionSpec`` object.
+#
+# If you're running this example in a Notebook, you can uncomment and execute the next cell to see what code is executed
+# on each data node.
+
+# %%
+
+# %load -s local_first_order_computation assets/functions/federated_analytics_functions.py
+
+# %%
+
+local_first_order_computation_docker_files = [
```

Comment:
```suggestion
local_first_order_computation_files = [
```

Reply:
Actually not sure about this proposition anymore. When I read it fast I read "dockerfiles" and not "docker_files". "docker_files" as files running on the docker container makes sense.

Reply:
I did not change it then